cirq-core/cirq/circuits/circuit.py

```diff
@@ -1950,7 +1951,27 @@ def transform_qubits(
         with _compat.block_overlapping_deprecation(re.escape(_DEVICE_DEP_MESSAGE)):
             return Circuit(op_list, device=self._device if new_device is None else new_device)
 
-    def _prev_moment_available(self, op: 'cirq.Operation', end_moment_index: int) -> Optional[int]:
+    def prev_moment_available(
+        self, op: 'cirq.Operation', *, end_moment_index: Optional[int] = None
+    ) -> int:
+        """Finds the index of the previous (i.e. left most) moment which can accommodate `op`.
```

Comment:
As Dax said, the name doesn't really suggest leftmost - I read it as "get the previous available moment", which seems to suggest _rightmost_. Maybe something like `earliest_equivalent_moment` instead?

Reply:
I think the word `equivalent` can also cause confusion. How about `earliest_available_moment` ?